web-ui/src/components/guild-results/GuildSummaryCard.jsx

```diff
@@ -134,22 +134,29 @@ const GuildSummaryCard = ({ guild, index, isOpen, onGuildSelect }) => {
     }
   };
 
-  const iconStyles = { height: '2.5rem', width: '2.5rem' };
+  const iconStyles = {
+    position: 'absolute',
+    bottom: '0.5rem',
+    right: '0.5rem',
+    height: '2.5rem',
+    width: '2.5rem'
+  };
+
+  const guildIcon = () => (
```

Comment:
I moved the JSX for the guild card icons into a new function to clean up the code.